src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -26,13 +27,13 @@
      * @param machines - list of machines to be filled with value
      * @param value    - any object of machine sub class
      */
-    void fill(List<Object> machines, Object value);
+    void fill(List<? super Machine> machines, Machine value);
```

Comment:
Якщо ми параметризували інтерфейс як `<T extends Machine>` то тепер у методах ми повинні відштовхуватись від меншої межі, тобто `T` а не від Machine. Інакше може виникнути ситуація, коли ми  інтерфейс параметризуємо наприклад треком, але при цьому в метод зможемо передати бульдозера. А по факту, якщо ми параметризовуємо треком, то метод повинен працювати тільки з треками і його наслідниками
це стосується всіх методів в інтерфейсі
```suggestion
    void fill(List<? super T> machines, T value);
```